src/praisonai/praisonai/cli/features/serve.py

```
@@ -289,11 +335,57 @@ async def invoke_agents(request: Request, query_data: AgentQuery = None):
+                                result = await agent.astart(query)
+                            elif hasattr(agent, 'start'):
+                                import asyncio
+                                loop = asyncio.get_event_loop()
+                                result = await loop.run_in_executor(None, agent.start, query)
+                            else:
+                                raise AttributeError(f"Agent {agent_name} has no start/astart method")
+                            return {"response": str(result)}
+                        else:
+                            raise HTTPException(status_code=404, detail=f"Agent '{agent_name}' not found")
+                    else:
+                        raise HTTPException(status_code=404, detail=f"Agent registry not available")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)